Create installing-tracking-script.md (#27)
mautic/user-documentation: docs/home/installing-tracking-script.md

## How to install the tracking script

> After installation and setup of the cron jobs you're ready to begin tracking contacts. You need to add a simple piece of JavaScript to the websites for each site you wish

Reviewer: ```suggestion
After installation and setup of the cron jobs you're ready to begin tracking Contacts. You need to add a piece of JavaScript to the websites for each site you wish
```

I think we should add a link to the cron jobs page here, also Contacts should be capitalised and we try not to use terms like 'simply' (Vale flags as a problem to fix)

Author: okay

> to track via Mautic. This is a very simple process and you can add this tracking script to your website template file,

Reviewer: We don't use terms like 'simple' - we use the Google Technical Writing Style Guide which explains more: https://developers.google.com/style/tone?hl=en

```suggestion
to track via Mautic. This is straightforward process, and you can add this tracking script to your website template file,
```

> or install a Mautic integration for the more common Content Management System platforms. Here is an example of the tracking JavaScript which you can access by clicking on 'Tracking Settings' in the global configuration:

Reviewer: ```suggestion
or install a Mautic integration for the more common Content Management System platforms. Here is an example of the tracking JavaScript which you can access by clicking on 'Tracking Settings' in the Global Configuration:
```

   ```
    (function(w,d,t,u,n,a,m){w['MauticTrackingObject']=n;
        w[n]=w[n]||function(){(w[n].q=w[n].q||[]).push(arguments)},a=d.createElement(t),
        m=d.getElementsByTagName(t)[0];a.async=1;a.src=u;m.parentNode.insertBefore(a,m)
    })(window,document,'script','https://example.com/mautic/mtc.js','mt');

    mt('send', 'pageview');
   ```

>    You should change the site URL,  `example.com/mautic` with the URL to your Mautic instance - in the  example provided, but it's recommended to copy

Reviewer: ```suggestion
   You should change the site URL,  `example.com/mautic` with the URL to your Mautic instance in the example provided, but it's recommended to copy
```

>    it from the tracking settings in your instance.

Reviewer: ```suggestion
   the whole code block from the tracking settings in your Mautic instance.
```
